.cdprc

```diff
@@ -37,25 +37,39 @@ fi
 
 FZF_VERSION=$(fzf --version | awk '{print $1}')
 if [[ "$FZF_VERSION" < "0.36.0" ]] ; then
-    echo "Problem: The installed program 'fzf' ${FZF_VERSION} is too old. At least version 0.36.0 is required."
-    echo "         If your distro doesn't provide a recent version you might want to look at"
-    echo "         https://github.com/junegunn/fzf/releases  for binary releases."
+    echo "Problem: The installed program 'fzf' ${FZF_VERSION} is too old."
+    echo "At least version 0.36.0 is required."
+    echo "If your distro doesn't provide a recent version you might want to look at"
+    echo "https://github.com/junegunn/fzf/releases  for binary releases."
     return 1
 fi
 
-echo -e "\n*** Define helpful functions and aliases:\n"
 unset FZFPOS
 
+RG="rg --type=markdown --no-ignore-parent"
+# Since arrays are not exportable(!?) as required for __list_files_with_status
```

Comment:
I never had to export an array and thus don't have any experience how to get this done best. But on [this page](https://stackoverflow.com/questions/5564418/exporting-an-array-in-bash-script) the topic was discussed and explained very detailed. Maybe this helps?